plugins/woocommerce/client/blocks/assets/js/base/components/quantity-selector/index.tsx

```diff
@@ -4,8 +4,8 @@
 import { __, sprintf } from '@wordpress/i18n';
```

Comment:
Changes in this file were to prevent submission happening too early when typing decimals. For example, if I wanted to type `2.5`, it would submit when I've typed `2`. The local state coupled with debounce fixes this.